src/test/java/org/dbos/apiary/CrossDBBenchmarkTests.java

```diff
@@ -0,0 +1,118 @@
+            conn.dropTable("ShopTransactions");
+            conn.createTable("ShopItems", "ItemID integer PRIMARY KEY NOT NULL, ItemName varchar(1000) NOT NULL, ItemDesc varchar(2000) NOT NULL, Cost integer NOT NULL, Inventory integer NOT NULL");
+            conn.createTable("ShopCart", "PersonID integer NOT NULL, ItemID integer NOT NULL, Cost integer NOT NULL");
+            conn.createTable("ShopOrders", "PersonID integer NOT NULL, OrderID integer NOT NULL, ItemID integer NOT NULL");
+            conn.createTable("ShopTransactions", "OrderID integer PRIMARY KEY NOT NULL, PersonID integer NOT NULL, Cost integer NOT NULL");
+            conn.createIndex("CREATE INDEX CartIndex ON ShopCart (PersonID);");
+        } catch (Exception e) {
+            logger.info("Failed to connect to Postgres.");
+        }
+        apiaryWorker = null;
+    }
+
+    @AfterEach
+    public void cleanupWorker() {
+        if (apiaryWorker != null) {
+            apiaryWorker.shutdown();
+        }
+    }
+
+    @BeforeEach
+    public void cleanupElasticsearch() {
+        try {
+            ElasticsearchClient client = new ElasticsearchConnection("localhost", 9200, "elastic", "password").client;
+            DeleteIndexRequest request = new DeleteIndexRequest.Builder().index("items").build();
+            client.indices().delete(request);
+        } catch (Exception e) {
+            logger.info("Index Not Deleted {}", e.getMessage());
+        }
+    }
+
+    @Test
+    public void testShopBenchmark() throws InvalidProtocolBufferException {
+        logger.info("testShopBenchmark");
+
+        ElasticsearchConnection conn;
+        PostgresConnection pconn;
+        try {
+            conn = new ElasticsearchConnection("localhost", 9200, "elastic", "password");
+            pconn = new PostgresConnection("localhost", ApiaryConfig.postgresPort, "postgres", "postgres", "dbos");
+        } catch (Exception e) {
+            logger.info("No Elasticsearch/Postgres instance! {}", e.getMessage());
+            return;
+        }
+
+        apiaryWorker = new ApiaryWorker(new ApiaryNaiveScheduler(), 4);
+        apiaryWorker.registerConnection(ApiaryConfig.elasticsearch, conn);
+        apiaryWorker.registerConnection(ApiaryConfig.postgres, pconn);
+        apiaryWorker.registerFunction("ShopAddItem", ApiaryConfig.postgres, ShopAddItem::new);
+        apiaryWorker.registerFunction("ShopSearchItem", ApiaryConfig.postgres, ShopSearchItem::new);
+        apiaryWorker.registerFunction("ShopAddCart", ApiaryConfig.postgres, ShopAddCart::new);
+        apiaryWorker.registerFunction("ShopCheckoutCart", ApiaryConfig.postgres, ShopCheckoutCart::new);
+        apiaryWorker.registerFunction("ShopGetItem", ApiaryConfig.postgres, ShopGetItem::new);
+        apiaryWorker.registerFunction("ShopESAddItem", ApiaryConfig.elasticsearch, ShopESAddItem::new);
+        apiaryWorker.registerFunction("ShopESSearchItem", ApiaryConfig.elasticsearch, ShopESSearchItem::new);
```

Comment:
These lines seem to be the same as the ones in the `ApiaryWorkerExecutable.java` file. Maybe we could just write one "BenchmarkSetup" function in utilities.